packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-account/wallet-account.tsx

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
 import { Icon, Text } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
 import './wallet-account.scss';
 
 type TWalletAccount = {
     balance: number;
     currency: string;
     name: string;
     icon: string;
+    is_mobile?: boolean;
 };
 
-const WalletAccount = ({ balance, name, currency, icon }: TWalletAccount) => {
+const WalletAccount = ({ balance, name, currency, icon, is_mobile = false }: TWalletAccount) => {
     return (
         <div className='wallet-account'>
             <Icon className='wallet-account__icon' icon={icon} size={24} />
             <div className='wallet-account__details'>
-                <Text as='div' color='prominent' size='xxs'>
+                <Text as='div' color='prominent' size={is_mobile ? 'xxs' : 'xs'} weight='bold'>
                     {name}
                 </Text>
-                <Text as='div' color='prominent' size='xxxs' weight='bold'>
-                    {balance} {currency}
+
+                <Text as='div' size={is_mobile ? 'xxxs' : 'xxs'}>
+                    {localize('Balance')}: {formatMoney(currency ?? '', balance, true)} {currency}
```

Comment:
@aum-deriv try to refactor it with Localize component
https://deriv-group.slack.com/archives/CU4NXCZGD/p1692838539666279?thread_ts=1692786200.111349&cid=CU4NXCZGD